### ISSUE TITLE
2.6.3 and 2.7 client with VFS in Suffix mode on win 10

### ISSUE THREAD

**Issue by voroyam** [owncloud/client#8125]
Install 2.6.3 client from our web site or the daily 2.7

https://download.owncloud.com/desktop/stable/ownCloud-2.6.3.14058.13619.msi

https://download.owncloud.com/desktop/ownCloud/daily/2.7/win/owncloudclient-2.7.0-daily20201002.2200-vfs.x64.msi

### Expected behaviour
The client should install and the VFS should be in the win...something mode

### Actual behaviour
Client installs and the VFS is in suffix mode

### Steps to reproduce
1. Get a win 10 VM
2. Download the client
3. Install the client with VFS

**Comment by voroyam** [comment]
```
10-02 12:53:19:908 [ info gui.socketapi ]:	Received SocketAPI message <-- "RETRIEVE_FILE_STATUS:C:\\Users\\dmitry\\ownCloud" from QLocalSocket(0x181295dfe80)
10-02 12:53:19:908 [ info gui.socketapi ]:	Sending SocketAPI message --> "STATUS:OK:C:\\Users\\dmitry\\ownCloud" to QLocalSocket(0x181295dfe80)
10-02 12:53:24:813 [ info gui.socketapi ]:	New connection QLocalSocket(0x18129b366b0)
10-02 12:53:24:813 [ info gui.socketapi ]:	Sending SocketAPI message --> "REGISTER_PATH:C:\\Users\\dmitry\\ownCloud" to QLocalSocket(0x18129b366b0)
10-02 12:53:24:813 [ info gui.socketapi ]:	Received SocketAPI message <-- "GET_STRINGS:CONTEXT_MENU_TITLE" from QLocalSocket(0x18129b366b0)
10-02 12:53:24:813 [ info gui.socketapi ]:	Sending SocketAPI message --> "GET_STRINGS:BEGIN" to QLocalSocket(0x18129b366b0)
10-02 12:53:24:813 [ info gui.socketapi ]:	Sending SocketAPI message --> "STRING:CONTEXT_MENU_TITLE:ownCloud" to QLocalSocket(0x18129b366b0)
10-02 12:53:24:813 [ info gui.socketapi ]:	Sending SocketAPI message --> "GET_STRINGS:END" to QLocalSocket(0x18129b366b0)
10-02 12:53:24:813 [ info gui.socketapi ]:	Received SocketAPI message <-- "GET_MENU_ITEMS:C:\\Users\\dmitry\\AppData\\Local\\Temp\\ownCloud-logdir\\20201002_1253_owncloud.log.0" from QLocalSocket(0x18129b366b0)
10-02 12:53:24:813 [ info gui.socketapi ]:	Sending SocketAPI message --> "GET_MENU_ITEMS:BEGIN" to QLocalSocket(0x18129b366b0)
10-02 12:53:24:813 [ info gui.socketapi ]:	Sending SocketAPI message --> "GET_MENU_ITEMS:END" to QLocalSocket(0x18129b366b0)
10-02 12:53:24:877 [ info gui.socketapi ]:	Lost connection  QLocalSocket(0x18129b366b0)
10-02 12:53:34:391 [ info gui.folder ]:	Trying to check "https://demetrio.owncloud.works/remote.php/dav/files/test2/" for changes via ETag check. (time since last sync: 58 s)
10-02 12:53:34:391 [ debug gui.folder.manager ]	[ OCC::FolderMan::slotRunOneEtagJob ]:	Scheduling "https://demetrio.owncloud.works/remote.php/dav/files/test2/" to check remote ETag
10-02 12:53:34:391 [ info sync.accessmanager ]:	6 "PROPFIND" "https://demetrio.owncloud.works/remote.php/dav/files/test2/" has X-Request-ID "107bdd8e-7377-43f0-9862-0587484ab85b"
10-02 12:53:34:391 [ info sync.httplogger ]:	"107bdd8e-7377-43f0-9862-0587484ab85b: Request: PROPFIND https://demetrio.owncloud.works/remote.php/dav/files/test2/ Header: { Depth: 0, Authorization: [redacted], User-Agent: Mozilla/5.0 (Windows) mirall/2.7.0daily20201002 (build 2200) (ownCloud, windows-10.0.14393 ClientArchitecture: x86_64 OsArchitecture: x86_64), Accept: */*, Content-Type: text/xml; charset=utf-8, X-Request-ID: 107bdd8e-7377-43f0-9862-0587484ab85b, } Data: [<?xml version=\"1.0\" ?>\n<d:propfind xmlns:d=\"DAV:\">\n  <d:prop>\n    <d:getetag/>\n  </d:prop>\n</d:propfind>\n]"
10-02 12:53:34:391 [ debug sync.cookiejar ]	[ OCC::CookieJar::cookiesForUrl ]:	QUrl("https://demetrio.owncloud.works/remote.php/dav/files/test2/") requests: (QNetworkCookie("oc_sessionPassphrase=sEJs3jWUooIX2wtQR5NAAw%2B5htVjgfvIKsKsFWiYIEahROPzBJK99xOB6iYr0CLlsszo0Xm3KVEfJKh31wYd%2Bg6KXrQxKBM7%2B3gxGBDoS40KkH23U5CSjuLMpQHhCF91; secure; HttpOnly; domain=demetrio.owncloud.works; path=/"), QNetworkCookie("ocnivdbdc8jv=vg5m0nqen72esfc92eej2i1kuk; secure; HttpOnly; domain=demetrio.owncloud.works; path=/"))
10-02 12:53:34:391 [ info sync.networkjob ]:	OCC::RequestEtagJob created for "https://demetrio.owncloud.works" + "/" "OCC::Folder"
```

**Comment by TheOneRing** [comment]
I'd need the first couple of lines when the client starts.

**Comment by voroyam** [comment]
looking for this?

10-02 12:57:56:109 [ warning plugins ]:	Plugin failed to load: "Die Bibliothek C:\\ownCloud\\owncloudsync_vfs_win.dll kann nicht geladen werden: Das angegebene Modul wurde nicht gefunden."

**Comment by voroyam** [comment]
```
10-02 12:57:56:045 [ debug gui.folderwatcher ]	[ OCC::FolderWatcher::pathIsIgnored ]:	* Ignoring file "C:/Users/dmitry/ownCloud/.owncloudsync.log"
10-02 12:57:56:109 [ warning plugins ]:	Plugin failed to load: "Die Bibliothek C:\\ownCloud\\owncloudsync_vfs_win.dll kann nicht geladen werden: Das angegebene Modul wurde nicht gefunden."
10-02 12:57:56:163 [ debug gui.folderwatcher ]	[ OCC::FolderWatcher::pathIsIgnored ]:	* Ignoring file "C:/Users/dmitry/ownCloud/.sync_c93845113264.db-wal"
10-02 12:57:56:164 [ warning default ]:	QFSFileEngine::open: No file name specified
10-02 12:57:56:242 [ info gui.folder.manager ]:	<========== Sync finished for folder [ownCloud] of account [demetrio.owncloud.works (test2)] with remote [https://demetrio.owncloud.works/remote.php/dav/files/test2/]
10-02 12:57:58:168 [ info gui.updater ]:	Checking for available update
10-02 12:57:58:168 [ info sync.accessmanager ]:	2 "" "https://updates.owncloud.com/client/?version=2.7.0.2200&platform=win32&oem=ownCloud&buildArch=x86_64&currentArch=x86_64&versionsuffix=daily20201002&channel=beta&msi=true" has X-Request-ID "53d23e4c-4d86-40db-874c-1baed91d1642"
10-02 12:57:58:168 [ info sync.httplogger ]:	"53d23e4c-4d86-40db-874c-1baed91d1642: Request: GET https://updates.owncloud.com/client/?version=2.7.0.2200&platform=win32&oem=ownCloud&buildArch=x86_64&currentArch=x86_64&versionsuffix=daily20201002&channel=beta&msi=true Header: { User-Agent: Mozilla/5.0 (Windows) mirall/2.7.0daily20201002 (build 2200) (ownCloud, windows-10.0.14393 ClientArchitecture: x86_64 OsArchitecture: x86_64), Accept: */*, X-Request-ID: 53d23e4c-4d86-40db-874c-1baed91d1642, } Data: []"
10-02 12:57:58:168 [ debug sync.cookiejar ]	[ OCC::CookieJar::cookiesForUrl ]:	QUrl("https://updates.owncloud.com/client/?version=2.7.0.2200&platform=win32&oem=ownCloud&buildArch=x86_64&currentArch=x86_64&versionsuffix=daily20201002&channel=beta&msi=true") requests: ()
10-02 12:57:58:246 [ info sync.httplogger ]:	"53d23e4c-4d86-40db-874c-1baed91d1642: Response: GET 200 https://updates.owncloud.com/client/?version=2.7.0.2200&platform=win32&oem=ownCloud&buildArch=x86_64&currentArch=x86_64&versionsuffix=daily20201002&channel=beta&msi=true Header: { Server: nginx, Date: Fri, 02 Oct 2020 10:57:58 GMT, Content-Type: text/xml;charset=UTF-8, Transfer-Encoding: chunked, Connection: keep-alive, Vary: Accept-Encoding, X-Powered-By: PHP/7.4.6, X-Frame-Options: SAMEORIGIN, X-XSS-Protection: 1; mode=block, X-Content-Type-Options: nosniff, Referrer-Policy: no-referrer-when-downgrade, Content-Security-Policy: default-src 'self' http: https: data: blob: 'unsafe-inline', Strict-Transport-Security: max-age=31536000; includeSubDomains; preload, Content-Encoding: gzip, } Data: []"
10-02 12:57:58:246 [ info gui.updater ]:	Version info arrived: Your version: 146085512912832664 Skipped version: 0 "" Available version: 0 "" Available version string: "" Web url: "" Download url: ""
10-02 12:57:58:246 [ info gui.updater ]:	No version information available at the moment
10-02 12:58:04:218 [ info gui.socketapi ]:	Received SocketAPI message <-- "RETRIEVE_FILE_STATUS:C:\\Users\\dmitry\\ownCloud" from QLocalSocket(0x16a68cf0ad0)
10-02 12:58:04:218 [ info gui.socketapi ]:	Sending SocketAPI message --> "STATUS:OK:C:\\Users\\dmitry\\ownCloud" to QLocalSocket(0x16a68cf0ad0)
```

**Comment by michaelstingl** [comment]
> * Get a win 10 VM

What exact VM? Ancient version? Minimum Win10 1709 is required.

**Comment by voroyam** [comment]
where can I see this version?

**Comment by TheOneRing** [comment]
open the start menu and type winver

**Comment by TheOneRing** [comment]
but the new user agent string does that for us already
windows-10.0.14393

**Comment by TheOneRing** [comment]
The plugin requires windows 1709 your running 1607 .
See https://en.wikipedia.org/wiki/Windows_10_version_history and https://docs.microsoft.com/en-us/windows/win32/api/cfapi/ns-cfapi-cf_process_info#requirements

**Comment by voroyam** [comment]
Great. I installed every update via windows update that was available, but still get the same error message.

**Comment by voroyam** [comment]
context of closing this issue. I had apparently an old VM although I downloaded it just some months ago from the official windows site.

I could update my current VM via complicated windows update process, or just download a new windows 10 ISO and make a new VM. In the new VM I had no issues with suffix mode, nor in my windows laptop. 

I wrote down what I have found in the docs issue I opened in the client repo for VFS
